### ISSUE TITLE
Error when using an empty ruleset

### ISSUE THREAD

**Issue by ngnijland** [sasstools/sass-lint#820]
**What version of Sass Lint are you using?**
1.8.2

**What did you do?**
I have the following code:

```sass
.class1
    color: #fff
    text-align: center

.class2

.class3
    color: #fff
    text-align: center
```

- .class2 has no properties.
- In the sass-lint config file, `no-empty-rulesets` is set to `1`.

**What did you expect to happen?**
I expected sass-lint to pass and give me a warning on the `no-empty-rulesets` rule.

**What actually happened?**
Sass-lint threw a fatal error:
`387:1  error  Please check validity of the block starting from line #387  Fatal`

Line 387 represents the line with `.class2`.

**Comment by DanPurdy** [comment]
this is a duplicate of #456
